TIGLViewer/src/TIGLViewerWindow.cpp

```diff
@@ -114,9 +116,20 @@ TIGLViewerWindow::TIGLViewerWindow()
 
     //cpacsConfiguration = new TIGLViewerDocument(this);
     scriptEngine = new TIGLScriptEngine(this);
-    
+
     setAcceptDrops(true);
 
+    // creator init
+    fuselageModificator->init();
```

Comment:
This was resolve at the same time as TIGLViewerWindow.ui  issue, because I need in every case to refactor the init function for the TIGLViewerWindow.ui  issue.